molgenis-questionnaires/src/main/frontend/src/store/getters.js

```diff
@@ -49,12 +49,7 @@ const isVisible = (field: ChapterField, formData: Object) => {
   try {
     visible = field.visible(formData)
   } catch (e) {
-    console.error(`Error in getters, during evaluation of function 'visible' for field ${field.id}.
-      Where visible expression is: 
-        '${field.visible.toString()}'
-      And formData is: 
-        '${JSON.stringify(formData)}'
-      The default value of visible is set to false`, e)
+    console.log(`Setting ${field.id}.visible to false.`)
```

Comment:
@sidohaakma is wondering why this is a console.log instead of a console.error

Reply:
When you start up the application with the vuex debugger open, you get about fifty of these messages which is overly disturbing. Therefore I toned down the red screaming messages a bit.

Reply:
ok, but the question remains: why log instead of error?

Reply:
Because it is not always an error. Talked with @sidohaakma and we'll turn it into a log.warn.